frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ManageServingRuntimeModal.tsx

```diff
@@ -138,6 +139,20 @@ const ManageServingRuntimeModal: React.FC<ManageServingRuntimeModalProps> = ({
       });
   };
 
+  const createNewToken = React.useCallback(() => {
```

Comment:
Yeah, great idea moving the addition logic from the component, it will be more reusable.